## 62% of end-end tests have passed
```
Total executed tests: 8
Failed tests: 3
Expected to fail tests: 0
Passed tests: 5
```
```
IDs of failed tests: 727230,727229,702843 
```
<h3>Failed tests (3)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=bring-back-advanced&PR_ID=21892&APK_NAME=21892.apk&TR_CASE_IDS=727230,727229,702843">Rerun failed tests</a></li><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3761219">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `EditBox` by `accessibility id`: `enter-contact-code-input` </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:316: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:409: in add_contact
    chat.public_key_edit_box.click()
../views/base_element.py:91: in click
    element.click()
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click
    self._execute(Command.CLICK_ELEMENT)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute
    return self._parent.execute(command, params)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element/00000000-0000-000c-ffff-ffff0000038f/click' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element/00000000-0000-000c-ffff-ffff0000038f/click' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/home/ltuser/.appium/node_modules/appium-uiautomator2-driver/node_modules/@appium/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at runMicrotasks (<anonymous>)
E       at processTicksAndRejections (node:internal/process/task_queues:96:5)
E       at AndroidUiautomator2Driver.click (/home/ltuser/.appium/node_modules/appium-uiautomator2-driver/lib/commands/element.js:159:3)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESqydcvU3VNUlWqJsM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8J9TctoMta7tnM4U">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3761222">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td>

```
Test setup failed: critical/test_wallet.py:32: in prepare_devices
    self.sender_username, self.receiver_username = self.home_1.get_username(), self.home_2.get_username()
../views/home_view.py:528: in get_username
    profile_view.default_username_text.wait_for_element(3)
../views/base_element.py:128: in wait_for_element
    raise TimeoutException(
 Device `1`: `Text` by` accessibility id`: `username` is not found on the screen after wait_for_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3761221">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Click until `Text` by `accessibility id`: `username` will be presented </div><div>Device 1: Find `ProfileButton` by `accessibility id`: `open-profile` </div></blockquote></p>

```
Test setup failed: critical/test_wallet.py:32: in prepare_devices
    self.sender_username, self.receiver_username = self.home_1.get_username(), self.home_2.get_username()
../views/home_view.py:528: in get_username
    profile_view.default_username_text.wait_for_element(3)
../views/base_element.py:128: in wait_for_element
    raise TimeoutException(
 Device `1`: `Text` by` accessibility id`: `username` is not found on the screen after wait_for_element
```

<br/><br/></td></tr></tbody></table></details><h3>Passed tests (5)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3761218">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESHIJaeVab5KrsgOig">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESTWBA3Nw4FMZipKP6">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3761217">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESC5bX9Wh86MmZ1P6M">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3761220">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESC5bX9Wh86MmZ1P6M">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3761223">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3761224">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr></tbody></table></details>

--- a/src/legacy/status_im/events.cljs
+++ b/src/legacy/status_im/events.cljs
@@ -7,6 +7,7 @@
     legacy.status-im.chat.models.loading
     legacy.status-im.data-store.chats
     legacy.status-im.data-store.visibility-status-updates
+    legacy.status-im.ens.core
     legacy.status-im.fleet.core
     legacy.status-im.group-chats.core
     legacy.status-im.log-level.core

--- a/src/status_im/subs/profile.cljs
+++ b/src/status_im/subs/profile.cljs
@@ -101,10 +101,22 @@
    public-key))
 
 (re-frame/reg-sub
+ :profile/light-client-enabled?
+ :<- [:profile/profile]
+ (fn [profile]
+   (get-in profile [:wakuv2-config :LightClient])))
+
+(re-frame/reg-sub
  :profile/store-confirmations-enabled?
  :<- [:profile/profile]
  (fn [profile]
    (get-in profile [:wakuv2-config :EnableStoreConfirmationForMessagesSent])))
+
+(re-frame/reg-sub
+ :profile/peer-syncing-enabled?
+ :<- [:profile/profile]
+ (fn [profile]
+   (:peer-syncing-enabled? profile)))
 
 (re-frame/reg-sub
  :profile/telemetry-enabled?

